controllers/event_rsvp.rb

```diff
@@ -1,7 +1,7 @@
 class CalendarEvents::RsvpController < ApplicationController
   attr_accessor :topic
   before_action :check_user_and_find_topic, only: [:add, :remove]
-  before_action :check_if_rsvp_enabled
+  before_action :check_if_rsvp_enabled, except: [:get_users]
```

Comment:
@fzngagan Why are we excepting get_users here?

Reply:
The reason being, `check_user_and_find_topic` sets the `@topic` property which is not called here. In the case of get_users, when `check_if_rsvp_enabled` is called, the `@topic` is not set and an error is raised.

Currently, I think excepting is the way to go.  Later, I'm thinking to plan and refactor this controller....

Reply:
Ok :+1: